src/test/java/com/solvd/micro9/scooters/integration/kstreams/KStreamsIT.java

```diff
@@ -0,0 +1,149 @@
+package com.solvd.micro9.scooters.integration.kstreams;
+
+import com.solvd.micro9.scooters.domain.Currency;
+import com.solvd.micro9.scooters.domain.RentEvent;
+import com.solvd.micro9.scooters.messaging.KStreamConfig;
+import com.solvd.micro9.scooters.messaging.serde.CustomSerde;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.common.serialization.Serializer;
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.TestInputTopic;
+import org.apache.kafka.streams.TestOutputTopic;
+import org.apache.kafka.streams.TopologyTestDriver;
+import org.apache.kafka.streams.state.KeyValueStore;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@SpringBootTest(classes = {KStreamConfig.class, StreamsBuilder.class})
+@DirtiesContext
+public class KStreamsIT extends KafkaTestcontainers {
```

Comment:
@eyakauleva
add messages to all your assertions
when the test will be failed, messages will be helpful for developers